pcsx2/gui/AppCoreThread.cpp

```diff
@@ -264,12 +264,23 @@ static int loadGameSettings(Pcsx2Config& dest, const Game_Data& game) {
 
 
 	if (game.keyExists("mvuFlagSpeedHack")) {
-		bool vuFlagHack = game.getInt("mvuFlagSpeedHack") ? 1 : 0;
+		bool vuFlagHack = !!game.getInt("mvuFlagSpeedHack");
```

Comment:
I'm not sure why you changed this, it will effect how the expression is evaluated.

If I"m reading this right; the original expression states that if the bool `vuFlagHack == 1` ; `mvuFlagSpeedHack` will also evaluate to `1` ; otherwise it will be `0`

Reply:
> I'm not sure why you changed this, it will effect how the expression is evaluated.

It's the same behaviour, nothing wrong here.

Reply:
oh ok.